ringpop_test.go

```diff
@@ -530,9 +530,8 @@ func (s *RingpopTestSuite) TestChecksum() {
 	s.Require().NoError(err, "unable to bootstrap single node cluster")
 
 	s.Equal(ready, s.ringpop.state)
-	checksum, err = s.ringpop.Checksum()
+	_, err = s.ringpop.Checksum()
```

Comment:
I prefer uncommenting `s.NotZero(checksum)`. Just tested it locally and tests seems to pass.